blocks/image-compare/editor.scss

```diff
@@ -0,0 +1,30 @@
+
+/* main block wrapper */
```

Comment:
I have perhaps been exposed too much to the strict Gutenberg principles we have here. Which is: Sentence case and end with a period. And use /* */ if you want the comment to be visible/baked into the CSS, and use // if you are just commenting for someone reading the SCSS file.

Reply:
Removed.